src/components/Coupon/index.jsx

```diff
@@ -0,0 +1,135 @@
+import React, { useState, useEffect } from "react";
+import { Typography, Card, Button } from "antd";
+import { useTranslation } from "react-i18next";
+
+import "./style.css";
```

Comment:
if you import this before antd, you can override it, just a tip! No need to change if you want